dev/hal_install_spin.docker

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+# Parameters of the GCP project Spinnaker is maintaining.
+ARG BUILD_PROJECT
+ARG BUILD_ACCOUNT
+ARG BUILD_KEY # json keyfile contents
```

Comment:
You can also load env vars in k8s from secrets, which would probably be the golden-path here. 

`  kubectl create secret generic kubeconfig --from-file=relative/path/to/kube/config`

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: hal
spec:
  containers:
    - name: hal
      image: hal
      env:
        - name: KUBE_CONFIG
          valueFrom:
            secretKeyRef:
              name: kubeconfig
              key: config # this is the name of the file you published in the secret
```